tests/convert-tests/027-large-symbol-link/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Make sure btrfs-convert can handle a symbol link which is 4095 bytes large
+
+source "$TEST_TOP/common" || exit
+source "$TEST_TOP/common.convert" || exit
+
+setup_root_helper
+prepare_test_dev 1G
+check_global_prereq mkfs.ext4
+
+link_target=""
+
+# Soft link in btrfs can only have inlined data extent.
+# So here we create a symbol link whose target is 4095 bytes length.
+for ((i = 0; i < 4095; i++)); do
+	link_target+="b"
+done
```

Comment:
You can do it in one go `printf "%0.sb" {1..4095}"`, this is already done in some other tests like mkfs-tests/014-rootdir-inline-extent/test.sh . This would be good to have in a helper so we don't have to figure it out each time, but that's for another patch.